addon/components/ember-popper-base.js

```diff
@@ -177,7 +187,16 @@ export default class EmberPopperBase extends Component {
       this._modifiers = modifiers;
       this._placement = placement;
 
-      this._popper = new Popper(popperTarget, popperElement, { eventsEnabled, modifiers, placement });
+      this._popper = new Popper(
+        popperTarget,
+        popperElement,
+        {
+          eventsEnabled,
+          modifiers,
+          placement,
+          onCreate: this.onCreate,
```

Comment:
We should cache these and check for changes like the other attrs.

Reply:
Not sure I follow you here... do you mean we should track the `onCreate` and `onUpdate` functions, and make a change cause `didChange` to be true, so a new popper instance is created? I what scenario would that be needed?

Reply:
That is correct. If a user changes the `onUpdate` or `onCreate` property, a new Popper instance would be needed to respect those new callbacks.